### PR TITLE
Make noarch

### PR DESCRIPTION
**Pull request by BastianZim**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Ref: https://github.com/conda-forge/conda-forge.github.io/pull/1839
Ref: https://github.com/conda-forge/poetry-feedstock/pull/77

### CODE REVIEW

**Comment by BastianZim** [comment]
@conda-forge-admin, please rerender

**Comment by BastianZim** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -51,16 +52,21 @@ requirements:
     - requests >=2.18,<3.0
     - requests-toolbelt >=0.9.1,<0.11.0
     - shellingham >=1.5,<2.0
-    - tomli >=2.0.1,<3.0.0  # [py<311]
+    - tomli >=2.0.1,<3.0.0
```
Comment (xylar) [inline recipe/meta.yaml:55]: This should be fine because of:
https://github.com/python-poetry/poetry/blob/1.3.1/src/poetry/utils/_compat.py#L11-L15

**Comment by xylar** [comment]
This looks good to me.  @BastianZim, mark it as ready for review when you like or just merge when you're ready.

File: recipe/meta.yaml (hunk 1 above)
Comment (BastianZim) [reply recipe/meta.yaml:55]: Ahh great, that was the only thing I wasn‘t sure of.

**Comment by BastianZim** [comment]
Great! I can‘t merge here, only staged-recipes 😄

**Review by xylar — APPROVED** [review]
Let's give this a try!

**Comment by xylar** [comment]
Ah, right.  Do you want to become a maintainer?  If not, no worries.

**Comment by BastianZim** [comment]
Happy to, thank you!

**Comment by BastianZim** [comment]
In #82 — thanks

**Pull request merged**